sycl/plugins/unified_runtime/ur/ur.hpp

```diff
@@ -315,3 +354,16 @@ class UrReturnHelper {
   void *param_value;
   size_t *param_value_size_ret;
 };
+
+// Global variables for ZER_EXT_RESULT_ADAPTER_SPECIFIC_ERROR
+constexpr size_t MaxMessageSize = 256;
+extern thread_local zer_result_t ErrorMessageCode;
+extern thread_local char ErrorMessage[MaxMessageSize];
+
+// Utility function for setting a message and warning
+[[maybe_unused]] void setErrorMessage(const char *message,
+                                      zer_result_t error_code);
+
+// Returns plugin specific error and warning messages
+// TODO: promote to Unified Runtime API
+zer_result_t zerPluginGetLastError(char **message);
```

Comment:
```suggestion
zer_result_t zerPluginGetLastError(char **message);

```

Reply:
what is the suggested change @steffenlarsen? cannot find the diff :)

Reply:
I assume it's the newline at the end-of-file, added.